src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -101,7 +102,7 @@ def import_config(cmd,
                                       depth=depth,
                                       content_type=content_type)
 
-        if not skip_features:
+        if strict or not skip_features:
```

Comment:
>strict or  [](http://example.com/codeflow?start=11&length=10)

So we cant skip features in strict mode?

Reply:
Oh I remember we discussed not supporting both options together because the usage will be ambiguous.